client/client.go

```diff
@@ -124,6 +124,19 @@ const (
 	//
 	// https://www.envoyproxy.io/docs/envoy/latest/operations/cli#cmdoption-concurrency
 	defaultConnectProxyConcurrency = "1"
+
+	// defaultTransparentProxyUID is the default UID of the Envoy proxy
+	// container user, for use with transparent proxy
+	defaultTransparentProxyUID = "101"
+
+	// defaultTransparentProxyOutboundPort is the default outbound port for the
+	// Envoy proxy, for use with transparent proxy. Note the default value
+	// patches the default TransparentProxy service default for
+	// OutboundListenerPort. If the cluster admin sets this value to something
+	// non-default, they'll need to update the metadata on all the nodes to
+	// match. See also:
+	// https://developer.hashicorp.com/consul/docs/connect/config-entries/service-defaults#transparentproxy
+	defaultTransparentProxyOutboundPort = "15001"
```

Comment:
I wouldn't mind seeing Envoy/Connect constants defined in 
https://github.com/hashicorp/nomad/blob/main/helper/envoy/envoy.go
(just convenient to have it all in one place)

Reply:
Oh perfect, I didn't realize we had that file. Will fix.